variables.tf

```diff
@@ -128,7 +117,7 @@ variable "container_filter" {
 }
 
 variable "agent_tags" {
-  description = "Map of tags to associate to all metrics that the agent collects. NOTE: Use the `add_cluster_name` boolean variable to add the cluster name as a tag."
+  description = "Map of tags to associate to all metrics that the agent collects. NOTE: Use the `add_cluster_name` boolean variable to add the cluster name as a tag, e.g `{'ibm-containers-kubernetes-cluster-name': 'cluster_name'}."
```

Comment:
this is a bad example to give - the cluster name tag is added by the `add_cluster_name` boolean